app/controllers/application_controller.rb

```diff
@@ -4,13 +4,24 @@ class ApplicationController < ActionController::Base
   helper_method :current_user, :logged_in?, :current_year, :admin?
 
   before_action :set_locale
+  before_action :restrict_pages, if: :json_request?
 
   def set_locale
     I18n.locale = cookies[:locale] || I18n.default_locale
   end
 
+  def restrict_pages
+    return if params[:page].to_i <= 100
+
+    render json: {}, status: :forbidden
```

Comment:
I think this should be an empty array `[]` rather than object

Reply:
@andrew Changed to `[]`.